README.md

```diff
@@ -1,2 +1,65 @@
+
 # Upstage Cookbook
-The Upstage Cookbook features examples using Upstage products. If you have cool examples, feel free to share them by submitting a PR to this repo.
+
+This cookbook aims to highlight innovative use cases and outputs created by leveraging Upstage Solar and Document AI’s capabilities. We aim to inspire others by showcasing the versatility and excellence of Upstage features across various domains. The Upstage cookbook is open to the community. If you have an interesting idea for using APIs, spot a typo, or want to add or improve a guide, feel free to contribute!
+
+### How to use?
+
+Get an API key from Upstage console to try examples in the cookbook. Set the environment variable `UPSTAGE_API_KEY` with your key, or create an .env file with the key.
+
+### Submission Guidelines
+
+**Content:**
+
+- Source code for apps, programs, or computational projects assisted by the Upstage APIs.
+- Tutorials, educational materials, or lesson plans developed with the Upstage APIs.
+- Any other interesting applications that showcase the Upstage API's potential!
+
+**Requirements:**
+
+- Code/text entries should be in .md, .ipynb or other common formats
+- Include an overview describing the project/output and its purpose. Provide step-by-step instructions in a logical order and Include any relevant tips, notes, or variations.
+- Projects should be original creations or properly credited if adapted from another source.
+- Submissions should not contain explicit violating content
+
+**Rights:**
+
+- You maintain ownership and rights over your submitted project
+- By submitting, you grant us permission to showcase and distribute the entry
+- Entries cannot infringe on any third-party copyrights or intellectual property
+
+Let us know if you need any other guidelines or have additional criteria to include! We’re happy to continue iterating on these submission rules.
+
+Disclaimer: The views and opinions expressed in community and partner examples do not reflect those of Upstage.
+  
+
+### API List
+
+| API | Description | Import | Example usage |
+| --- | --- | --- | --- |
+| Chat | Build assistants using Solar Mini Chat | from langchain_upstage import ChatUpstage | [Link](https://python.langchain.com/v0.1/docs/integrations/chat/upstage/) |
+| Text Embedding | Embed strings to vectors | from langchain_upstage import UpstageEmbeddings | [Link](https://python.langchain.com/v0.1/docs/integrations/text_embedding/upstage/) |
+| Groundedness Check | Verify groundedness of assistant's response | from langchain_upstage import UpstageGroundednessCheck | [Link](https://python.langchain.com/v0.1/docs/integrations/tools/upstage_groundedness_check/) |
+| Layout Analysis | Serialize documents with tables and figures | from langchain_upstage import UpstageLayoutAnalysisLoader | [Link](https://python.langchain.com/v0.1/docs/integrations/document_loaders/upstage/) |
```

Comment:
Column 3, 4 are LangChain-specific. I suggest to remove them and add link to our developer's documentation.